Implement Accept Alert tests for WPT wdspec tests

alchiem opened the pull request:
<!-- Reviewable:start -->

<!-- Reviewable:end -->

alchiem committed:
Implement Accept Alert tests for WPT wdspec tests

w3c-bots commented:
# Build PENDING

Started: None
Finished: None


<h2>Failing Jobs</h2>
<ul>

<li>firefox:nightly</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18564)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/275668426)

alchiem commented:
Testing webdriver accept alert spec.
http://www.w3.org/TR/webdriver/#accept-alert

alchiem committed:
Add tests for alert, confirm and prompt dialogs.

alchiem committed:
Update comments

alchiem committed:
Deleted unnecessary import inline

andreastt commented on webdriver/tests/user_prompts/accept_alert.py:
```diff
@@ -0,0 +1,44 @@
+
+def test_accept_prompt(session):
+    # 18.2 step 3
+    session.execute_script("window.prompt(\"Enter Your Name: \", \"Federer\");")
```
Accepting a prompt will return an empty string, whereas dismissing it returns null. You should set this variable on the window global and then verify that the prompt is actually getting accepted.

alchiem replied:
fixed

andreastt reviewed (changes requested):
This looks mostly good. Just one change needed. Please fix up the commits as well.

alchiem committed:
Validate the value returned by confirm and prompt dialogs

alchiem commented:
Hi @andreastt,

What do you mean by fix up the commits as well?

Thanks,
-Alex

andreastt commented:
> What do you mean by fix up the commits as well?

Sorry, I should’ve explained this in more detail.

I want you to squash your commits together into one with a good commit message. This branch currently has five commits: the original “Implement Accept Alert tests” followed by four fixup commits. You can fold the latter four commits into the first commit.

andreastt commented:
@alchiem Disregard that. I was able to do it for you through the GitHub UI.

alchiem commented:
Hey @andreastt, 

So I added the same changes from dismiss alert to this branch where I moved the assert_success before the assert on the result of the dialog. Do I make another branch for that since you merged in the changes already?

andreastt commented:
@alchiem Ah yes, I missed that here. If you would file another PR to fix that, I would be most obliged.
